tutor/src/flux/course-enrollment-handled-errors.coffee

```diff
@@ -0,0 +1,13 @@
+ERROR_MAP =
+  invalid_enrollment_code: 'The provided enrollment code is not valid. Please verify the enrollment code and try again.'
+  already_enrolled: 'You are already enrolled in this course.  ' +
+                    'Please verify the enrollment code and try again.'
+  multiple_roles: 'We currently do not support teacher accounts with multiple associated student enrollments.'
+  dropped_student: 'You have been dropped from this course. Please speak to your instructor to rejoin.'
+  already_processed: 'Your enrollment in this course has been processed. Please reload the page.'
+  already_approved: 'Your enrollment in this course has been approved. Please reload the page.'
+  already_rejected: 'Your request to enroll in this course has been rejected for an unknown reason. Please contact OpenStax support.'
+  taken: 'The provided student ID has already been used in this course. Please try again or contact your instructor.'
+  blank_student_identifier: 'The student ID field cannot be left blank. Please enter your student ID.'
+
+module.exports = ERROR_MAP
```

Comment:
Not saying you should do that here, but if the strings got much longer you could use coffee script heredoc.

```coffee
already_enrolled: '''
  You are already enrolled in this course. 
  Please verify the enrollment code and try again.
'''
```

Just wanted to make sure you knew about 'em.